client/my-sites/google-my-business/select-location/index.js

```diff
@@ -109,11 +122,7 @@ export default connect(
 			},
 			{
 				id: 67890,
-				address: [
-					'234 Piedmont Drive',
-					'Talihassee, FL 34342',
-					'USA',
-				],
+				address: [ '234 Piedmont Drive', 'Talihassee, FL 34342', 'USA' ],
```

Comment:
Curious, there are quite a few places where code was reformated like this in this pull request. Is it because of some code editor's settings, or because of Prettier?

Reply:
This is because of my editor. I had a mental model where any formatting done by the editor would be un-done by the pre commit script, which clearly isn't the case.